src/kernel/init.c

```diff
@@ -604,6 +604,7 @@ void vm_exit(u8 code)
 
     tuple root = get_root_tuple();
     if (root) {
+        // expected_exit_code
```

Comment:
I wouldn't add these comments, I don't think they make the code easier to read, they are just duplicating the `sym()` string that is in the next line.